src/WebJobs.Script.WebHost/Controllers/AdminController.cs

```diff
@@ -120,16 +121,25 @@ public IHttpActionResult GetHostStatus()
                     };
                 }
 
+                _traceWriter.Info($"Host Status: {JsonConvert.SerializeObject(status, Formatting.Indented)}");
+
                 return Ok(status);
             }
             else
             {
-                // for Anonymous requests, we don't return any info
-                return Ok();
+                return Unauthorized();
             }
         }
 
         [HttpPost]
+        [Route("admin/host/ping")]
```

Comment:
Talking with Ahmed, we realized that the previous overloading of the host/status endpoint for truly anonymous requests would break him. E.g. he uses admin key, and expects that if he sends a key value that is invalid, host/status will return 401. So I'm adding back a separate ping endpoint.